linode/manager
# feat: [M3-7526] - Add setHeader to GET /profile endpoint

## Description 📝
In order for the proxy account to access the username of the parent account, we need to use the auth token of the parent to collect the parent's profile info on behalf of the proxy account. We will have the parent account's token available to us for this use after storing it in local storage when the parent logs in as the proxy user via the "Switch Account" button.

## Changes  🔄
List any change relevant to the reviewer.
- Added setHeaders to the /profile endpoint
- Added an empty object to all useProfile instances

## How to test 🧪
- Add the code from the commit named [use this to test PR](https://github.com/linode/manager/pull/9987/commits/33ccb670d37f85f3e2afa6b331f39c56f04396aa) - make sure to remove the `undefined` param from the `useProfile` call
- Make sure to add a PAT from another account to your .env file (this could be your personal account or a test account)
- Ensure both usernames are logged similar to the screenshot below
![Screenshot 2023-12-13 at 5 35 25 PM](https://github.com/linode/manager/assets/139489745/f5ec2ebe-b0a0-4ce4-bd35-a39bf4bd3340)

## As an Author I have considered 🤔

*Check all that apply*
